windows/src/global/inc/keymankeyboard_id.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ *
+ * Name:             KeymanKeyboardIID
+ * Description:      Header file to include the keyman keyboard GUIDs
+ */
+#ifndef _keymankeyboard_id_h
+#define _keymankeyboard_id_h
+
+//const c_clsidKMTipTextService: TGUID = '{FE0420F1-38D1-4B4C-96BF-E7E20A74CFB7}';  // version 10.0
+//const c_clsidKMTipTextService_90: TGUID = '{487EB753-DB93-48C5-9E6A-4398E777C61D}';   // I3663   // I4248
+//const c_clsidTextServicesFramework: TGUID = '{529A9E6B-6587-4F23-AB9E-9C7D683E3C50}';
```

Comment:
I found these other GUIDs mentioned in `utiltsf.pas`  however they are not referenced anywhere else.  Does this mean I should just define the main one `c_clsidKMTipTextService` as I have already done?

Reply:
Yeah, the others won't be needed here at present -- one is for an old version of the Keyman TSF addin, and the other is the CLSID for the Text Services Framework itself, which is available already if we need it.